cumulusci/core/config/project_config.py

```diff
@@ -139,11 +139,11 @@ def _validate_package_api_format(self):
 
     @property
     def config_global_local(self):
-        return self.global_config_obj.config_global_local
+        return self.universal_config_obj.config_global_local
 
     @property
     def config_global(self):
-        return self.global_config_obj.config_global
+        return self.universal_config_obj.config_global
```

Comment:
I think we should also rename `config_global_local` to `config_global`, and `config_global` to `config_universal`, for consistency. It's not backwards-compatible, but should be a pretty easy adjustment to make in downstream code.

Reply:
And actually, given this and other subtle backwards incompatibilities, I'm now thinking maybe it's better to _not_ keep the backwards-compatible BaseGlobalConfig alias.

Reply:
Can you confirm that the names in quotes here are not really used for anything and can/should also be changed?

https://github.com/SFDO-Tooling/CumulusCI/blob/master/cumulusci/core/config/project_config.py#L117

Reply:
Or if they are used for something, you know what to change in the other apps?

Reply:
Just used for error reporting it seems...